scene/resources/default_theme/default_theme.cpp

```diff
@@ -143,7 +143,7 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 
 	// Font Colors
 
-	Color control_font_color = Color(0.88, 0.88, 0.88);
+	Color control_font_color = Color(0.55, 0.55, 0.55);
```

Comment:
Not very familiar with the code here, but won't this affect all/a lot of text, not just the tooltips in the audio bus editor?